### ISSUE TITLE
css whitelist doesnt work with spaces

### ISSUE THREAD

**Issue by TDA** [leizongmin/js-xss#37]
## Steps to reproduce
```
var xss = require('xss');
var options = {
  whiteList: {
    a: ['style']
  },
  css: {
    whiteList: {
      'color': true,
      'text-decoration': true
    }
  }
}
var html = "<a style = 'color: #0095dd; text-decoration: none;'>xss</a>";
var myxss = new xss.FilterXSS(options)
console.log(myxss.process(html))
```

## Expected Behaviour:
The code is valid, and is not sanitized.

## Actual behaviour:
The style value is completely removed, even though both color and text-decoration are valid.

If i remove the spaces between `style` and `=`, then it works as expected.

**Comment by leizongmin** [comment]
Because this is not a strict HTML format, I will fix this problem, but it will takes some time.

**Comment by TDA** [comment]
@leizongmin Thank you very much!

**Comment by leizongmin** [comment]
I have published a new version `xss@0.2.4`, the following HTML can works well:

```html
<img alt = "example" width= 100px height = "200">
```

you can try the new version, and if any question please let me know.

**Comment by TDA** [comment]
@leizongmin Thank u so much! :)
